packages/kotti-ui/source/kotti-field/KtField.vue

```diff
@@ -32,7 +32,10 @@
 				class="kt-field__help-description"
 				v-text="field.helpDescription"
 			/>
-			<div class="kt-field__input-container-wrapper">
+			<div
+				v-show="!field.isLoadingValue"
```

Comment:
why the v-show?

Reply:
KtFieldDates have hooks that break with `v-if`